tests/kafkatest/sanity_checks/test_mirror_maker.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ducktape.tests.test import Test
+from ducktape.utils.util import wait_until
+
+from kafkatest.services.zookeeper import ZookeeperService
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.console_consumer import ConsoleConsumer
+from kafkatest.services.verifiable_producer import VerifiableProducer
+from kafkatest.services.mirror_maker import MirrorMaker
+
+
+class TestMirrorMakerService(Test):
+    """Sanity checks on console consumer service class."""
+    def __init__(self, test_context):
+        super(TestMirrorMakerService, self).__init__(test_context)
+
+        self.topic = "topic"
+        self.zk1 = ZookeeperService(test_context, num_nodes=1)
+        self.zk2 = ZookeeperService(test_context, num_nodes=1)
+
+        self.k1 = KafkaService(test_context, num_nodes=1, zk=self.zk1,
+                                  topics={self.topic: {"partitions": 1, "replication-factor": 1}})
+        self.k2 = KafkaService(test_context, num_nodes=1, zk=self.zk2,
+                                  topics={self.topic: {"partitions": 1, "replication-factor": 1}})
+
+        self.num_messages = 1000
+        # This will produce to source kafka cluster
+        self.producer = VerifiableProducer(test_context, num_nodes=1, kafka=self.k1, topic=self.topic,
+                                           max_messages=self.num_messages, throughput=1000)
+        self.mirror_maker = MirrorMaker(test_context, sources=[self.k1], target=self.k2, whitelist=self.topic)
+
+        # This will consume from target kafka cluster
+        self.consumer = ConsoleConsumer(test_context, num_nodes=1, kafka=self.k2, topic=self.topic,
+                                        consumer_timeout_ms=10000)
+
+    def setUp(self):
+        # Source cluster
+        self.zk1.start()
+        self.k1.start()
+
+        # Target cluster
+        self.zk2.start()
+        self.k2.start()
+
+    def test_lifecycle(self):
```

Comment:
How valuable are tests like this when we have the one below it that just adds the producing + consuming? I don't want to bloat the runtime for the tests if a test isn't really adding value. So do we think that if we did mess up something and this test failed that the issue wouldn't be obvious in the other test? Presumably we would just see the `start()` or `stop()` call failing with some timeout?